services/blockchain-connector/methods/tests/decoder.js

```diff
@@ -14,56 +14,55 @@
  *
  */
 const {
```

Comment:
Can we please update the filenames to be appropriate?

Reply:
Applied.